backend/src/apiserver/model/pipeline_version.go

```diff
@@ -29,20 +32,24 @@ const (
 )
 
 type PipelineVersion struct {
-	UUID           string `gorm:"column:UUID; not null; primary_key"`
+	gorm.Model
```

Comment:
What is `gorm.Model` for?

Reply:
`gorm` allows to enforce migration logic rules that will be very helpful. For example, we can enforce `belongs to` as shown [here](https://gorm.io/docs/belongs_to.html).